engine/execution/state/bootstrap/bootstrap_test.go

```diff
@@ -42,7 +43,7 @@ func TestBootstrapLedger(t *testing.T) {
 
 		expectedStateCommitment := unittest.GenesisStateCommitment
 
-		if !assert.Equal(t, expectedStateCommitment, stateCommitment) {
+		if !assert.Equal(t, fmt.Sprint(expectedStateCommitment), fmt.Sprint(stateCommitment)) {
```

Comment:
For whatever reason, the diffs shown by failures are now printed as hex-encoding of the ASCII representation, which makes it hard to find the original value and replace it with the new one.
`Sprint`-ing it restores the old behaviour of showing here e.g. `ac2eaefb594337486722fd11a64fc465038790822b646ddd8264c4a8595c9a61` vs `ef864c71406b34e7937f56cbbf69b6f58bd27f796070def9034411e7a818d727` as such